src/control/jobs/control_jobs.c

```diff
@@ -1143,18 +1143,20 @@ static int32_t dt_control_local_copy_images_job_run(dt_job_t *job)
 
   dt_tag_new("darktable|local-copy", &tagid);
 
+  gboolean tag_change = FALSE;
   while(t && dt_control_job_get_state(job) != DT_JOB_STATE_CANCELLED)
   {
     const int imgid = GPOINTER_TO_INT(t->data);
     if(is_copy)
     {
-      if (dt_image_local_copy_set(imgid) == 0)
-        dt_tag_attach_from_gui(tagid, imgid, FALSE, FALSE);
+      if(dt_image_local_copy_set(imgid) == 0)
+      {
+        if(dt_tag_attach(tagid, imgid, FALSE, FALSE)) tag_change = TRUE;
```

Comment:
`dt_tag_attach` already returns bool so wouldn't this be ok to change to:
```c
tag_change = dt_tag_attach(tagid, imgid, FALSE, FALSE) || tag_change;
```?

Reply:
yes this is similar.
Thought I think the previous version is more readable and doesn't change the code size.
Is there any gain using OR and not IF ?

Reply:
Essentially miniscule. for me "OR" version is more readable. in theory the "OR" version should be tiny bit more performant, but since this isn't pipe processing or anything like that the difference would be at most jump+cache loss due to incorrect prediction.